modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-azure.adoc

```diff
@@ -0,0 +1,183 @@
+
+image::{dir}/azure-jupyter-console-10.PNG[Config Azure web app,align="center" width=75%]
+
+* Copy the following URI in the corresponding field
+** `https://github.com/Teradata/lake-demos.git`
+* Click Clone
+
+image::{dir}/azure-jupyter-console-clone-11.PNG[Config Azure web app,align="center" width=75%]
+
+* On the Jupyter Lab console click in the Lake Demos folder
+
+image::{dir}/azure-jupyter-console-click-lake-demos-12.PNG[Config Azure web app,align="center" width=75%]
+
+image::{dir}/azure-jupyter-console-lakedemos-13.PNG[Config Azure web app,align="center" width=75%]
+
+=== Find the IP of the Jupyter Lab extensions Azure Web App
+* On JupyterLab open a notebook with Teradata Python kernel and run the following command for finding your notebook instance IP address.
+
++
+[source, python, id="lakedemos_azure_first_config", role="emits-gtm-events, content-editable"]
+----
+import requests
+def get_public_ip():
+    try:
+        response = requests.get('https://api.ipify.org')
+        return response.text
+    except requests.RequestException as e:
+        return "Error: " + str(e)
+my_public_ip = get_public_ip()
+print("My Public IP is:", my_public_ip)
+----
+
+** We will whitelist this IP in your VantageCloud Lake environment in order to allow the connection
+** This is for purposes of this guide and the notebooks demos. For production environments, a more robust networking setting might be needed
+** Azure App Service offers, as well, a list of all possible IP addresses that the service might expose. This is under the overview tab
+
+image::{dir}/azure-app-service-ips-14.PNG[Loaded JupyterLab,align="center" width=75%]
+
+== VantageCloud Lake Configuration
+* In the VantageCloud Lake environment, under settings, add the IP of your notebook instance
+** A lake environment supports multiple address whitelisting
+
+image::{auxdir}/sagemaker-lake.PNG[Initiate JupyterLab,align="center" width=75%]
+
+== Jupyter Notebook Demos for VantageCloud Lake
+
+=== Configurations
+* The file https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file] should be edited to match the configuration of your VantageCloud Lake environment +
+
+image::{auxdir}/sagemaker-vars.PNG[Initiate JupyterLab,align="center" width=75%]
+
+* Especifically the following values should be added 
+
++
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+
+| *"host"* 
+| Public IP value from your VantageCloud Lake environment
+
+| *"UES_URI"* 
+| Open Analytics from your VantageCloud Lake environment
+| *"dbc"*
+| The master password of your VantageCloud Lake environment
+|====
+
++
+IMPORTANT: Remember to change all passwords in the vars.json file.
+
+* You'll see that in the sample vars.json, the passwords of all users are defaulted to "password", this is just for illustration purposes, you should change all of these password fields to strong passwords, secure them as necessary, and follow other password management best practices.
```

Comment:
Perhaps it makes more sense to switch order here? That way we offer the context first and then enforce the message with the callout?

```suggestion
* You'll see that in the sample vars.json, the passwords of all users are defaulted to "password", this is just for illustration purposes, you should change all of these password fields to strong passwords, secure them as necessary, and follow other password management best practices.

IMPORTANT: Remember to change all passwords in the vars.json file.
```